src/docs/asciidoc/http-client.adoc

```diff
@@ -429,6 +429,40 @@ public class Application {
 ----
 ====
 
+
+== Bootstrap Configuration 
+
+Use `HttpClient.from` and create a `TcpClient` when you need more control of your client configuration, i.e., when you 
+want to specify the local address.
+
+
+====
+[source,java]
+----
+import reactor.netty.http.client.HttpClient;
+import reactor.netty.tcp.TcpClient;
+import java.net.InetSocketAddress;
+
+public class Application {
+
+    public static void main(String[] args) {
+	    TcpClient tcpClient = TcpClient.create()
+					.bootstrap(b -> b.localAddress(new InetSocketAddress("host", port)));
```

Comment:
@lfmunoz Yesterday we made a change in order to make the API more obvious when you what to add this configurations (See https://github.com/reactor/reactor-netty/commit/5e2364ce139b78b8b96dceb9b3bb2d2efec11aa2)

Can you adapt the code snippet to 
```
TcpClient tcpClient = TcpClient.create()
  			       .bindAddress(new InetSocketAddress("host", port));
```